webapp/app/translations/de/LC_MESSAGES/messages.po

```diff
@@ -494,115 +494,57 @@ msgstr ""
 "ELSTER” können Privatpersonen die für die Einkommensteuererklärung "
 "notwendigen Formulare ausfüllen und abschicken."
 
-#: app/forms/steps/eligibility_steps.py:331
-#: app/forms/steps/eligibility_steps.py:421
+#: app/forms/steps/eligibility_steps.py:328
+#: app/forms/steps/eligibility_steps.py:430
 msgid "form.eligibility.user_a_has_elster_account.yes"
 msgstr "Ja"
 
-#: app/forms/steps/eligibility_steps.py:332
-#: app/forms/steps/eligibility_steps.py:422
+#: app/forms/steps/eligibility_steps.py:329
+#: app/forms/steps/eligibility_steps.py:431
 msgid "form.eligibility.user_a_has_elster_account.no"
 msgstr "Nein"
 
-#: app/forms/steps/eligibility_steps.py:343
+#: app/forms/steps/eligibility_steps.py:336
+#: app/forms/steps/eligibility_steps.py:412
+msgid "form.eligibility.elster_account_failure-error"
+msgstr ""
+"Für die Überprüfung Ihrer Identität braucht unser Service einen "
+"Freischaltcode. Dieser Freischaltcode wird von ELSTER erstellt und per "
+"Post versendet. Aus technischen Gründen geschieht das allerdings nur, "
+"wenn Sie noch kein Konto bei Mein ELSTER haben. Zurzeit können Sie den "
+"Steuerlotsen daher nicht nutzen.</p><p>Es gibt bereits Überlegungen, den "
+"Steuerlotsen so weiterzuentwickeln, dass er auch genutzt werden kann, "
+"wenn bereits ein Konto bei Mein ELSTER vorhanden ist."
+
+#: app/forms/steps/eligibility_steps.py:346
 msgid "form.eligibility.user_b_has_elster_account-title"
 msgstr ""
 "Hat die Person, mit der Sie zusammen veranlagen möchten, ein Konto bei "
 "Mein ELSTER?"
 
-#: app/forms/steps/eligibility_steps.py:349
+#: app/forms/steps/eligibility_steps.py:352
 msgid "form.eligibility.user_b_has_elster_account.yes"
 msgstr "Ja"
 
-#: app/forms/steps/eligibility_steps.py:350
+#: app/forms/steps/eligibility_steps.py:353
 msgid "form.eligibility.user_b_has_elster_account.no"
 msgstr "Nein"
 
-#: app/forms/steps/eligibility_steps.py:357
+#: app/forms/steps/eligibility_steps.py:360
 msgid "form.eligibility.divorced_joint_taxes_failure-error"
 msgstr ""
 "Der Steuerlotse bildet die Einzelveranlagung für geschiedene Paare "
 "zurzeit nicht ab."
 
-#: app/forms/steps/eligibility_steps.py:401
+#: app/forms/steps/eligibility_steps.py:404
 msgid "form.eligibility.alimony.single.yes"
 msgstr "Ja, ich zahle oder beziehe Unterhalt."
 
-#: app/forms/steps/eligibility_steps.py:402
+#: app/forms/steps/eligibility_steps.py:405
 msgid "form.eligibility.alimony.single.no"
 msgstr "Nein, weder zahle, noch beziehe ich Unterhalt."
 
-#: app/forms/steps/eligibility_steps.py:429
-msgid "form.eligibility.elster_registration_method_failure-error"
-msgstr ""
-"Für die Überprüfung Ihrer Identität braucht unser Service einen "
-"Freischaltcode. Dieser Freischaltcode wird von ELSTER erstellt und per "
-"Post versendet. Aus technischen Gründen geschieht das allerdings in Ihrem"
-" Fall nicht. Daher können Sie den Steuerlotsen derzeit nicht nutzen."
-
```

Comment:
Mh, I don't think you meant to remove that?

Reply:
I think it is still there and was only moved to l. 508?

Reply:
Very good catch 👍